src/main/java/com/hierynomus/smbj/share/DiskShare.java

```diff
@@ -190,6 +211,38 @@ public FileInfo getFileInformation(SMB2FileId fileId) throws SMBApiException, Tr
     }
 
     /**
+     * Get Share Information for the current Disk Share
+     * @return the ShareInfo
+     * @throws SMBApiException
+     */
+    public ShareInfo getShareInformation() throws TransportException, SMBApiException {
+    	final String path = "";
+
+    	if ( shareInfo != null && attrExpiration > System.currentTimeMillis() ) {
```

Comment:
Please us the same code-style as the rest of the project. `if (shareInfo` no space between the `(` and the check